index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+    <nav id="mobile-navbar">
+        <div class="nav-items welcome-logo">Welcome</div>
+        <div class="nav-items">
+            <img src="./assets/icons/hamburger-menu-24.png" alt="hamburger-menu-icon">
+        </div>
+    </nav>
+
+    <section id="main" class="main-section">
+        <div class="main-content-margin-div text-center">
+            <h2 id="header-first" class="primary-header-text color-deep-blue">Hey there. I’m Mariane</h2>
+            <h2 id="header-second" class="primary-header-text color-theme-green">I’m a software developer</h2>
+        </div>
+        <div class="main-content-margin-div text-center">
+            <p class="heading-supporting-text color-deep-blue">
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="main-content-margin-div text-center">
+            <div class="social-media">
+                <a href="#">
+                    <img src="./assets/icons/social-icons/twitter.png" alt="Twitter">
+                </a>
+                <a href="#">
+                    <img src="./assets/icons/social-icons/Linkedin-icon.png" alt="LinkedIn">
+                </a>
+                <a href="#">
+                    <img src="./assets/icons/social-icons/medium.png" alt="Medium">
+                </a>
+                <a href="#">
+                    <img src="./assets/icons/social-icons/github.png" alt="GitHub">
+                </a>
+                <a href="#">
+                    <img src="./assets/icons/social-icons/angellist.png" alt="Angel list">
+                </a>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] The button shown below is missing in your design. Please refer to the image and apply the necessary changes to your code 👍 
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/164273352-e535eb68-f5c6-492a-9dab-b34c365ec7ea.png)

Reply:
@codepantha Added the button, but this part not getting marked as 'outdated' after fix like other points by git.
Can you please suggest me something on this matter?

Reply:
You don't have to worry about it. Next time, just make your changes, commit, and ask for a review once you're satisfied with your changes 👍

Reply:
@codepantha Thanks a ton for the clarification. Will keep it in mind. Sorry, I am new to this PR process flow and its features so I was confused.